src/System.Reflection.Emit/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.3.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
If it is targeting netstandard and shipped in its independent package we probably should bump the version but if it is netcoreapp only then we don't need to bump the version as it will only ship with the big flat netcore.app package.